account_move_line_stock_info/hooks.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def post_init_hook(cr, registry):
+
+    """ INIT stock_move_id references in acount move line from account_move"""
+    # FOR stock moves
+    cr.execute(
+        """
+        UPDATE account_move_line aml
+            SET stock_move_id = am.stock_move_id
+        FROM account_move am
+        WHERE am.id = aml.move_id AND am.stock_move_id IS NOT NULL;
```

Comment:
```suggestion
        WHERE am.id = aml.move_id AND am.stock_move_id IS NOT NULL AND aml.stock_move_id IS NULL;
```

Reply:
Ok. Thanks for the explanation.

Regarding my suggestion, `AND aml.stock_move_id IS NULL` it is just a suggestion. I understand the column will be null for most cases. But imagine someone has filled the column using a custom script/module. You will not want the column overwritten in that case. I think it is not harmful to add the condition.

Reply:
Thank you so much, I applied your suggestion  ;)